gdcdictionary/schemas/clinical_trials.yaml

```diff
@@ -157,26 +126,14 @@ properties:
 
   clinical_trials:
     description: >
-      Clinical Trials
-    type: array
-    items:
-      type: string
-
-  participants:
-    description: >
-      Participants
+      Indication of clinical trial numbers associated with a study.
     type: array
     items:
       type: string
 
-  template_link:
-    description: >
-      Template link
-    type: string
-
   fda_approved:
     description: >
-      FDA approved
+      The current state of the study (FDA approved).
     type: string
```

Comment:
I think this should be an enum of "Yes" and "No"

Reply:
This can be done.  The records will have to be manually curated as most are in the following form: "FDA-approved since 2010, approved to treat various type of arthritis, including rheumatoid arthritis, and cytokine release syndrome".
Unknown and NA will also need to be added as enums.

Reply:
Also, I could change the property name to "status" to be more in line with FHIR.  Thoughts?